tests/broker-engine/services-and-bulk-stmt.robot

```diff
@@ -296,14 +297,16 @@ RLCode
 
     ${INITIAL_SCRIPT_CONTENT}    Catenate
     ...    function init(params)
-    ...    broker_log:set_parameters('/tmp/toto.log', 2)
+    ...    broker_log:set_parameters('/tmp/test4.log', 2)
     ...    end
     ...
     ...    function write(d)
     ...    broker_log:info(0, "toto")
     ...    return true
     ...    end
```

Comment:
This is a lua script. Could you respect indentations?
we declare two functions, it would be cool to indent correctly the code inside the functions...

Reply:
done